plugins/mesh/src/gltf/glTFFileLoader.cpp

```diff
@@ -49,25 +48,19 @@ bool megamol::mesh::GlTFFileLoader::getGltfDataCallback(core::Call& caller) {
     CallGlTFData* gltf_call = dynamic_cast<CallGlTFData*>(&caller);
     if (gltf_call == NULL) return false;
 
-    checkAndLoadGltfModel();
+    auto has_update = checkAndLoadGltfModel();
+    if (has_update) {
+        ++m_version;
+    }
 
-    gltf_call->setData(m_gltf_model);
+    if (gltf_call->version() < m_version) {
+        gltf_call->setData(m_gltf_model,m_version);
+    }
```

Comment:
Also found this pattern in the WavefrontObjLoader. I don't think the if cond. is needed. I also don't think it is needed withing setData, as long as pointer sized values are stored in calls.

I think it might be a leftover pattern from back when setData was simply incrementing the version internally and the version wasn't explicitly set.